pkg/ddc/goosefs/transform_hadoop_config.go

```diff
@@ -1,3 +1,17 @@
+/*
```

Comment:
Please follow the license header sample [here](https://github.com/fluid-cloudnative/fluid/blob/master/pkg/ddc/base/interface.go#L1-L15) to add empty lines.